### ISSUE TITLE
Connection not available with VPN and Lulu running on Big Sur

### ISSUE THREAD

**Issue by nnenci** [objective-see/LuLu#252]
Hi and thank you very much for this lovely and useful piece of software.
I have updated to Bug Sur (no clean installation) and realized that Lulu creates a connection that appears among the Network connections on MacOS Settings.
When Lulu is active and I turn on my IPSec VPN (I use the default system VPN Cisco IPSec service), the computer is not able to get any internet connection.
Perhaps this one of mine is a silly question, but I have not figured out how to solve this problem.
Any help would be very welcome.
Thank you in advance.
All best.

**Comment by RzzPla** [comment]
hi , i have the same issue, i've installed big sur and lulu 2.0 ( first try with upgrade, second try i unistalled old version and install 2.0) .  everithing works fine, but when i connect my mac by Vpn ip sec to work's networks, i can't ping any adresses. 
i deleted all roules but nothing change. i authorize all apple program but nothing change. if i disable lulu all pings start to respond. any help or workaround are welcome. 
thanks 
P.

**Comment by bpsys** [comment]
Same here, also upgrade (no clean install) to Big Sur but I had to remove the old LuLu Version to install 2.0. As soon as I connect to an IPSec VPN I am not able to use Microsoft Remote Desktop or even browse the web interface of an vCenter located in the corporate network. Thanks in advance!

**Comment by Tummetott** [comment]
I can confirm than Cisco IPSec VPNs are not working with Lulu 2.0.0 under Catalina.

**Comment by dmzh22** [comment]
I seem to have the same problem.

Can connect to NordVPN when Lulu is running.
Can not connect to CiscoVPN (university) when Lulu is running.

MacOS Catalina 10.15.7 _ Lulu 2.2 just updated.

**Comment by objective-see** [comment]
Thanks for the bug (reports).

tl;dr Apple bug, fixed by them in macOS 11.1 

Unfortunately this an issue in Apple's Network Extension framework ...that affects any product/tool that leverages the framework :(  From another vendor (that leveraged the Network Extension framework) : 

"_There is a bug in macOS ...where VPNs ... won’t work when a Network Extension is installed. This affects ... many other products based on Network Extensions. The VPN will connect successfully, but no data is transferred. The bug has been reported to Apple_" 

Apple has fixed this (and other Network Extension framework bugs) in macOS 11.1 ...so the (only?) solution is to update to latest version of Big Sur!

**Comment by manuelhubacher** [comment]
I’m running into the same problem: As long as LuLu is running I can’t connect to my University's IPSec VPN.

- macOS 11.4
- LuLu 2.3.1

**Comment by nstefi** [comment]
I work on my office PC remotely connected to it using the Microsoft Remote Desktop app on Mac. I was getting disconnected very frequently, almost every 1/2 hour to 1 hr.
First I thought is from the firewall, but I checked with SonicWall and they said everything is fine and other users who use Windows don't get disconnected.
I noticed that if I quit Lulu I could work long time, I wasn't getting disconnected, so it seems that the issue must be from Lulu on the Mac.
Lulu version: 2.2.0
macOS version: 12.0.1 (Monterey)
VPN client is SonicWall Mobile Connect, version 5.0.8
Microsoft Remote Desktop App version: 10.7.0
